best-practices/tech_lead.md

```diff
@@ -0,0 +1,313 @@
+
+It helps to know who you are, what your skills and talents are, and to ask for help when you need it.
+
+## Project Management Has to Be Done
+
+... and team members are happier if it is done well.
+
+### It's About Process
+There needs to be a transparent process that:
+- documents the work cycle and sprint goals for easy lookup (e.g epics, milestones)
+- enables devs to easily choose the highest priority work for the sprint and work cycle (e.g ready column)
+- keeps all devs actively engaged in working towards sprint and work cycle goals (e.g. population of ready column, easy way for devs working on a ticket to surface problems and get resolution quickly)
+- makes it easy for the Tech Lead (TL), the Product Owner (PO) and management to see which work is in progress and how it relates to the goals (the board as a whole)
+
+If you're TL, and the work-cycle PO is mostly unavailable, and there's no appointed scrum master ... project management still needs to be done.  You don't have to do everything yourself ... but you need to ensure it all gets done.
+
+There's also engineering management opportunities when you are TL:
+- Which tech debt should we be paying?
+- “How should CDL fit into ETD work cycle?”
+- “Can the team be more-or-less single focused on ETD work for x weeks, then be single focused on Sinopia afterwards?”
+
+Ask the questions, start the discussions that will help the work-cycle and the team be as successful and happy as possible.
+
+### The Board the Board the Board
+We've been using "project boards" for transparency of process during workcycles.  More on them below.
+
+### Keep Track of Action Items Somewhere
+
+Action Items may come up in a variety of contexts:
+- during standup / discussions after standup
+- during meetings between TL and PO, or any other subset of the team
+- during full team meetings (planning, retro)
+- conversations during pairing ...
+- planned and impromptu meetings
+
+For ETD 2020Q3 work cycle, many action items ultimately became tickets, but they often started out as text in the running notes document, assigned to an individual with a comment.  Sometimes the action item was "create a ticket about" because writing the ticket needed a few more minutes than were available at the time.
+
+### Working with the PO
+
+Each TL and each PO and each project is different.  The "right" amount of time for the TL and the PO to confer depends on individual styles, on the actual work being done for the work cycle, the active work in the moment, etc.
+
+Keep a running list of topics to be discussed -- the running notes are a good place for this nascent "agenda" for a TL + PO meeting.
+
+- meet as often as you need to, or if that's not possible, as often as you can.
+  - for ETD 2020Q3, there were so many details and questions, the TL and the PO met for 30 min every day
```

Comment:
See above comments about specific dates/projects in this kind of documentation.

Reply:
I do agree that the project info here feels out of place. How about something like `we have found that a daily check-in between the TL and PO is an effective practice for managing the many details and questions that need to be clarified before tickets can be actionable`.... That could certainly use some word smithing but I think it clarifies the reason for the check ins.

Reply:
finessed the wording - thank you.